
syntax = "proto2";

package mesop.components.slider;

message SliderType {
  optional bool disabled = 1;
  optional bool discrete = 2;
  optional bool show_tick_marks = 3;
  optional double min = 4;
  optional string color = 5;
  optional bool disable_ripple = 6;
  optional double max = 7;
  optional double step = 8;

  optional string on_value_change_handler_id = 9;
}
